doc/uftrace-report.md

```diff
@@ -152,6 +155,17 @@ This command shows information like the following:
        26.046 us    0.939 us           1  a
        25.107 us    0.934 us           1  b
 
+    $ uftrace report --srcline
```

Comment:
This example actually doesn't show source location because the target program is not recorded with `--srcline` and `abc` is may not be compiled with `-g`.

So I think we better write the example as follows:
```suggestion
    $ uftrace record --srcline abc
    $ uftrace report --srcline
```

Reply:
I agree. Thank you.